distribution/src/bin-filemode-755/hz-start

```diff
@@ -22,15 +22,15 @@ readJvmOptionsFile "jvm.options"
 
 #TODO -Djet.home= ????
 JAVA_OPTS_ARRAY=(\
-$JDK_OPTS \
+"$JDK_OPTS" \
 "-Dhazelcast.logging.type=log4j2" \
 "-Dlog4j.configurationFile=file:$HAZELCAST_HOME/config/log4j2.properties" \
 "-Djet.home=$HAZELCAST_HOME" \
```

Comment:
Could you check if this is still used, possibly remove the TODO above...

Reply:
Removed the property & TODO and updated the PR description. Also removed one more unused jet property.

Reply:
Removed IMDG scripts as well.